packages/ignite-cli/src/extensions/ignite/addModule.js

```diff
@@ -23,10 +23,14 @@ module.exports = (plugin, command, context) => {
     // install the module
     if (useYarn) {
       const addSwitch = options.dev ? '--dev' : ''
-      await system.run(`yarn add ${moduleFullName} ${addSwitch}`)
+      const cmd = `yarn add ${moduleFullName} ${addSwitch}`
+      ignite.log(cmd)
+      await system.spawn(cmd, { stdio: 'ignore' })
```

Comment:
It's not safe though. It's `ignore`.  I have no clue what `react-native link` is doing that would cause this, and I've looked. 😆 

Now that we have `spawn`, `exec` and `run`, I do agree this needs order.

That was my intention behind `run` originally.  Maybe we switch to `run` and `runAdvanced`?  

`run` would just return `stdout`.  And `runAdvanced` could return an object like `spawn` does if you need extra details?